packages/shared/lib/clients/provider.client.ts

```diff
@@ -192,6 +200,70 @@ class ProviderClient {
         }
     }
 
+    private async createStripeAppToken(tokenUrl: string, code: string, clientSecret: string, callback: string): Promise<object> {
+        try {
+            const headers = {
+                'Content-Type': 'application/x-www-form-urlencoded',
+                Authorization: 'Basic ' + Buffer.from(clientSecret + ':').toString('base64')
+            };
+            const body = {
+                grant_type: 'authorization_code',
+                code: code,
+                redirect_uri: callback
+            };
+
+            const response = await axios.post(tokenUrl, body, { headers: headers });
+            if (response.status === 200 && response.data) {
+                return {
+                    access_token: response.data['access_token'],
+                    livemode: response.data['livemode'],
+                    refresh_token: response.data['refresh_token'],
+                    scope: response.data['scope'],
+                    stripe_publishable_key: response.data['stripe_publishable_key'],
+                    stripe_user_id: response.data['stripe_user_id'],
+                    token_type: response.data['token_type'],
+                    expires_in: 3600
+                };
+            }
+
+            throw new NangoError('stripe_app_token_request_error');
+        } catch (e: any) {
+            throw new NangoError('stripe_app_token_request_error', e.message);
+        }
+    }
+
+    private async refreshStripeAppToken(refreshTokenUrl: string, refreshToken: string, clientSecret: string): Promise<object> {
+        try {
+            const headers = {
+                'Content-Type': 'application/x-www-form-urlencoded',
+                Authorization: 'Basic ' + Buffer.from(clientSecret + ':').toString('base64')
+            };
+
+            const body = {
+                grant_type: 'refresh_token',
+                refresh_token: refreshToken
+            };
+
+            const response = await axios.post(refreshTokenUrl, body, { headers: headers });
+
+            if (response.status === 200 && response.data) {
+                return {
+                    access_token: response.data['access_token'],
+                    livemode: response.data['livemode'],
+                    refresh_token: response.data['refresh_token'],
+                    scope: response.data['scope'],
+                    stripe_publishable_key: response.data['stripe_publishable_key'],
+                    stripe_user_id: response.data['stripe_user_id'],
+                    token_type: response.data['token_type'],
+                    expires_in: 3600
```

Comment:
Can we make this a const and put this at the top of the file in the event it ever changes we can easily change it?

Reply:
Sure, I'll refactor both methods by placing `const expires_in` at the top of the file. This way, it will be easier to locate and modify if needed in the future. Thank you.